webpack.config.js

```diff
@@ -52,7 +52,7 @@ const config = {
       // Enable CSS Modules for Skyportal
       {
         test: /\.css$/,
-        include: /static\/js/,
+        include: [/static\/js/, /node_modules\/react-datepicker\/dist/],
```

Comment:
Why is this necessary?

Reply:
This is needed for this import line: `import 'react-datepicker/dist/react-datepicker-cssmodules.css';`